t/simulators/vpcd.c

```diff
@@ -253,6 +254,18 @@ _spew(int lines)
         _spew_one(i);
 }
 
+static int is_bad_plug(int n)
+{
+    if (bad_plugs_count) {
+        int i;
+        for (i = 0; i < bad_plugs_count; i++) {
+            if (bad_plugs[i] == n)
+                return 1;
+        }
+    }
+    return 0;
+}
+
```

Comment:
Would an array of flags be better here?  Then instead of a function call do
```c
if (bad_plugs[i % NUM_PLUGS]...)
```

Reply:
it's a good idea, tried it out, but ended up keeping what I had.  I think adding the flags makes the code a tad more confusing for a future developer.  I think storing the actual plug makes things more obvious.

After seeing your comment, I even contemplated making the bad_plugs a bitmask.  But figured that was overkill.